dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -285,3 +285,15 @@ CWPKI2061E.useraction=Ensure that the certificate is a valid X.509 certificate.
 CWPKI2062E=CWPKI2062E: The {0} OCSP responder URL defined in the server configuration is not a valid URI. If defined, it must be a valid URI to override the OSCP responder URL contained in the certificate.
 CWPKI2062E.explanation=Certificate revocation checking requires a valid OCSP responder URL.
 CWPKI2062E.useraction=Provide a valid OCSP responder URL in the server configuration.
+
+CWPKI2063E=CWPKI2063E: The ACME Service failed to automatically renew the certificate with the {0} serial number. The certificate is revoked. The renew request error is ''{1}''. The request is scheduled to try again in {2}.
+CWPKI2063E.explanation=The ACME Service tried to renew a certificate but encountered an error. The ACME Service continues to request a new certificate until a new certificate is issued. SSL/TLS requests fail until a new certificate request is successful.
+CWPKI2063E.useraction=Review the error message for details on the failure. 
+
+CWPKI2064I=CWPKI2064I: The certificate with the {0} serial number is revoked. The ACME service will request a new certificate from the ACME certificate authority at the {1} URI.
+CWPKI2064I.explanation=The ACME Service detected that the certificate was revoked and automatically requests a new certificate.
```

Comment:
**change**
The ACME Service detected that the certificate was revoked and automatically requests a new certificate.

**to**
The ACME Service detected that the certificate was revoked and automatically requested a new certificate.
**or**
he ACME Service detects that the certificate was revoked and automatically requests a new certificate.